store/store_test.go

```diff
@@ -327,6 +329,36 @@ func TestStore_EnsureIndexes(t *testing.T) {
 	}
 }
 
+func TestStore_prometheusMetrics_indexes(t *testing.T) {
```

Comment:
These `_test.go` files are only executed as part of the unit tests, so they would never appear in the production environment - ie - no Prometheus. Unless you're planning on plumbing the CI environment through somehow?

Reply:
They were just unit tests that ensure that the metric exists and are described/collected properly. They were not collecting any data. Although they are deleted now due to Todd's remarks.